TokenSale.sol

```diff
@@ -48,14 +57,16 @@ contract TokenSaleInterface {
 
 
 contract TokenSale is TokenSaleInterface, Token {
-    function TokenSale(uint _minValue, uint _closingTime, address _privateSale) {
+    function TokenSale(uint _minValue, uint _closingTime, address _privateSale) 
+    {
         closingTime = _closingTime;
         minValue = _minValue;
         privateSale = _privateSale;
     }
 
     function buyTokenProxy(address _tokenHolder) returns (bool success) {
-        if (now < closingTime && msg.value > 0 && (privateSale == 0 || privateSale == msg.sender)) {
+        if (now < closingTime && msg.value > 0 && (privateSale == 0 
```

Comment:
If you split an if then put each conditional in its own line. But in this case since the second condition is an `OR` parenthesized combination please move the entire inner parentheses to the second line.